src/main/scala/wdl4s/AstTools.scala

```diff
@@ -286,6 +286,26 @@ object AstTools {
 
   def terminalMap(ast: Ast, source: WdlSource) = (findTerminals(ast) map {(_, source)}).toMap
 
+  def wdlSectionToStringMap(ast: Ast, node: String, wdlSyntaxErrorFormatter: WdlSyntaxErrorFormatter): Map[String, String] = {
+    ast.findAsts(node) match {
+      case a if a.isEmpty => Map.empty[String, String]
+      case a if a.size == 1 =>
+        // Yes, even 'meta {}' and 'parameter_meta {}' sections have RuntimeAttribute ASTs.
+        // In hindsight, this was a poor name for the AST.
```

Comment:
Looks like the offender is:
```
     $kv = :identifier :colon $e -> RuntimeAttribute(key=$0, value=$2)
```
... which is causing all the key/value pairs to create RuntimeAttribute ASTs. You could rename it something more generic like `KeyValuePair`, and update all wdl4s builders to search for that instead of `RuntimeAttribute`. Hopefully...